recipes/cfanalytics/meta.yaml

```diff
@@ -14,12 +14,13 @@ source:
 build:
   number: 0
   skip: True  # [py<36]
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
   build:
-    - python
-    - setuptools
+    - python 3.6
```

Comment:
You might want `python >=3.6` (so that the package will still be good when 3.7 comes out)